src/Management/test/Endpoint.Test/Metrics/MetricsEndpointTest.cs

```diff
@@ -28,9 +28,9 @@ public MetricsEndpointTest(ITestOutputHelper output)
     [Fact]
     public void Constructor_ThrowsIfNulls()
     {
-        Assert.Throws<ArgumentNullException>(() => new MetricsEndpoint(null, null));
+        Assert.Throws<ArgumentNullException>(() => new MetricsEndpoint(null, null, null));
         IOptionsMonitor<MetricsEndpointOptions> options = GetOptionsMonitorFromSettings<MetricsEndpointOptions>();
-        Assert.Throws<ArgumentNullException>(() => new MetricsEndpoint(options, null));
+        Assert.Throws<ArgumentNullException>(() => new MetricsEndpoint(options, null, null));
```

Comment:
Personally, I don't see the point of having these tests for constructor-injected types. However, this test now looks incomplete, because it does not handle the case where all parameters are non-null, except `logger`. So I propose to either add that case, or remove the test altogether.

Maybe this is something to talk about with the rest of the team?

Reply:
I agree for now I'll add the test to make it consistent.